contracts/cosmwasm-vm/cw-ibc-core/src/ics04_channel/packet/receive_packet.rs

```diff
@@ -276,64 +328,12 @@ impl<'a> CwIbcCoreContext<'a> {
 
-                let chan_end_on_b = self.get_channel_end(deps.storage, &port_id, &channel_id)?;
-                cw_println!(deps, "execute_receive_packet decoding of data successful");
-                let packet_already_received = match chan_end_on_b.ordering {
-                    // Note: ibc-go doesn't make the check for `Order::None` channels
-                    Order::None => false,
-                    Order::Unordered => self
-                        .get_packet_receipt(deps.storage, &port_id, &channel_id, seq.into())
-                        .is_ok(),
-                    Order::Ordered => {
-                        let next_seq_recv =
-                            self.get_next_sequence_recv(deps.storage, &port_id, &channel_id)?;
-
-                        // the seq_on_a number has already been incremented, so
-                        // another relayer already relayed the packet
-                        seq < Into::<u64>::into(next_seq_recv)
-                    }
-                };
-
-                cw_println!(deps, "before packet already received ");
-                if packet_already_received {
-                    return Ok(Response::new().add_attribute("message", "Packet already received"));
-                }
-
-                cw_println!(deps, "before channel ordering check");
-
-                // `recvPacket` core handler state changes
-                match chan_end_on_b.ordering {
-                    Order::Unordered => {
-                        self.store_packet_receipt(
-                            deps.storage,
-                            &port_id,
-                            &channel_id,
-                            seq.into(),
-                            Receipt::Ok,
-                        )?;
-                    }
-                    Order::Ordered => {
-                        self.increase_next_sequence_recv(deps.storage, &port_id, &channel_id)?;
-                    }
-                    _ => {}
-                }
-                cw_println!(deps, "before after channel ordering check");
-
-                let event_recieve_packet = create_packet_event(
-                    IbcEventType::ReceivePacket,
-                    to_raw_packet(packet.clone()),
-                    chan_end_on_b.ordering(),
-                    &chan_end_on_b.connection_hops[0],
-                    None,
-                )?;
-
-                cw_println!(deps, "event recieve packet: {:?}", event_recieve_packet);
+                let channel_end = self.get_channel_end(deps.storage, &port_id, &channel_id)?;
 
                 let mut res = Response::new()
                     .add_attribute("action", "channel")
                     .add_attribute("method", "execute_receive_packet")
-                    .add_attribute("message", "success: packet receive")
-                    .add_event(event_recieve_packet);
+                    .add_attribute("message", "success: packet receive");
 
```

Comment:
Removal off callback missing?

Reply:
ya that also seems to be on current issue. I should have done this on current one.